src/admin/redux/modules/letters.js

```diff
@@ -0,0 +1,57 @@
+const { createAction } = require('redux-actions')
+const { get_body } = require('app/http')
+const { merge, compose, objOf, map, props, zipWith, pick } = require('ramda')
+const formatDate = require('app/format_date')
+
+const SEND_NEWSLETTER_POST =
+  'SEND_NEWSLETTER_POST'
+const SEND_SUB_REMINDER_POST =
+  'SEND_SUB_REMINDER_POST'
+
+const initialState = {
+  post_members: [],
+  sub_reminders: []
+}
+
+const reducer = (state = initialState, { type, payload }) => {
+  switch (type) {
+  case SEND_NEWSLETTER_POST:
+    return { ...state, post_members: payload.results }
+  case SEND_SUB_REMINDER_POST:
+    const idObj = map(pick([ 'id' ]), payload.results)
+    const addressArray = map(addressArr, payload.results)
+    const contentArray = map(objOf('email_content'), map(inject, payload.results))
+    const letterObj = zipWith(merge, contentArray, addressArray)
```

Comment:
This is looking so good and Ramda-ish, guys!

My only concern is that we're mapping over `payload.results` a lot -- it would be nice to do that in one go.

A first step might be to replace line 23 with:
`map(compose(objOf('email_content'), inject), payload.results)`

Another step might be to replace the expression for `sub_reminders` with
`reduce(zipWith(merge), {}, [idObj, contentArray, addressArray])` and remove `letterObj`.


Then we might consider using `R.converge` or (better) `R.liftN` to abstract over the `map`s